pom.xml

```diff
@@ -2,8 +2,40 @@
    <modelVersion>4.0.0</modelVersion>
    <groupId>com.rapid7.client</groupId>
    <artifactId>dcerpc</artifactId>
+   <packaging>jar</packaging>
    <version>0.0.2-SNAPSHOT</version>
    <name>dcerpc</name>
+   <url>https://www.rapid7.com</url>
+   <description>DCE-RPC implementation capable of using SMBv2 via SMBJ to invoke remote procedure calls (RPC) over the IPC$ named pipe.</description>
+
+   <developers>
+      <developer>
+         <id>pmiseiko</id>
+         <name>Paul-Andrew Joseph Miseiko</name>
+         <email>pmiseiko@rapid7.com</email>
+         <url>https://www.rapid7.com</url>
+      </developer>
+   </developers>
+
+   <license>
```

Comment:
This needs to be "licenses" as It is throwing error when running the build.

[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]   
[ERROR]   The project com.rapid7.client:dcerpc:0.0.2-SNAPSHOT (/home/travis/build/rapid7/smbj-rpc/pom.xml) has 1 error
[ERROR]     Malformed POM /home/travis/build/rapid7/smbj-rpc/pom.xml: Unrecognised tag: 'license' (position: START_TAG seen ...</developers>\n\n   <license>... @20:13)  @ /home/travis/build/rapid7/smbj-rpc/pom.xml, line 20, column 13 -> [Help 2]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.